src/ralph/api/auth/__init__.py

```diff
@@ -1,12 +1,11 @@
 """Main module for Ralph's LRS API authentication."""
 
-from ralph.api.auth.basic import get_authenticated_user as get_basic_user
-from ralph.api.auth.oidc import get_authenticated_user as get_oidc_user
+from ralph.api.auth.basic import get_basic_auth_user
+from ralph.api.auth.oidc import get_oidc_user
```

Comment:
I found that using aliases was making the code and dev process more complicated than necessary, which is why I changed this. 

Since there are currently no constraints for homogeneity (no base class), I would argue we are not really breaking anything here. We could make the auth methods more homogenous but this would require a bit more work which should probably be handled in a separate PR.

Reply:
closed by thumbsup